aws_cloudtrail_rules/aws_manual_securtygroup_changes.yml

```diff
@@ -0,0 +1,295 @@
+AnalysisType: rule
+Filename: aws_manual_securtygroup_changes.py
```

Comment:
Typo in file name `aws_manual_securtygroup_changes` -> `aws_manual_securitygroup_changes` (this is the name of the file so tests are working, but both these files should be renamed).